deployments/compose/docker-compose.yml

```diff
@@ -0,0 +1,142 @@
+version: "3"
+services:
+
```

Comment:
It's not everything without spark :)

Reply:
Spark!

Reply:
Added spark with 3 workers - note I didn't configure it to make use of lakeFS as a gateway or the underlying storage to be MINIO.